apps/discussions/contracts/DiscussionApp.sol

```diff
@@ -45,12 +45,12 @@ contract DiscussionApp is IForwarder, AragonApp {
         post.author = msg.sender;
         post.postCid = postCid;
         post.discussionThreadId = discussionThreadId;
-        post.createdAt = now;
+        post.createdAt = now; // solium-disable-line security/no-block-members
```

Comment:
Just for future reference, we are using the TimeHelpers from AragonOS for these cases, so you can perfectly replace `now` for `getTimestamp()` or `getTimestamp64()` depending on what data type is `post.createdAt` (uint or uint64), but these kind of changes will likely will be applied on future audits to these contracts, so no worries! just to share the info :)

NOTE: The danger will be there anywhere, but it should be prevented at contract design stage, by avoiding using block members for critical outcomes, since these values are gameable by miners

Source: https://github.com/aragon/aragonOS/blob/next/contracts/common/TimeHelpers.sol#L36
(TimeHelpers commands are automatically inherited by every AragonApp contract, and `now` is just an alias to `block.timeStamp`)

Reply:
> avoiding using block members for critical outcomes

Good to know about `getTimestamp` for AragonOS! In this case, I don't think the time at which a comment is created is a _critical outcome_, so I doubt it matters, but if I'd mistaken on that point I'd love for you to correct me!

Reply:
Yeah I mean they're functionally identical, the main advantage is in testing since you can create [a mock that overrides that call](https://github.com/aragon/aragon-apps/blob/master/apps/finance/contracts/test/mocks/FinanceMock.sol).